Codigo/src/file-coverage/models/file-coverage.ts

```diff
@@ -34,12 +35,43 @@ export class FileCoverage {
       textEditor,
       this.lcovFiles
     );
-
+    
     const coverageLines = this.lcovFilesToCoverageLines(lcovFiles);
+
     if (coverageLines) {
       return coverageLines;
     }

```

Comment:
Após o conflito, acho que fiz o merge errado. Toque essa condicional por:
```suggestion
    if (!coverageLines) {
      return new CoverageLines();
    }

```
e na linha 75 colocar o `return coverageLines;`